scripts/performance/perf_load/perf_processes.py

```diff
@@ -448,6 +453,73 @@ def check_genesis(gen_path):
         shutil.rmtree(dir_to_dlt, ignore_errors=True)
 
 
+def demote_promote(gen_path, interval):
```

Comment:
Why don't we send it like a common request sending in load tests?

Reply:
Reworked according to discussion.